tests/dashboards/verifyGCRemoteInstance_test.js

```diff
@@ -0,0 +1,170 @@
+    remoteInstancesPage.waitUntilRemoteInstancesPageLoaded();
+    remoteInstancesPage.openAddRemotePage(instanceType);
+    await remoteInstancesPage.addRemoteDetails(instanceDetails);
+    I.click(remoteInstancesPage.fields.addService);
+    pmmInventoryPage.verifyRemoteServiceIsDisplayed(instanceDetails.serviceName);
+    await pmmInventoryPage.verifyAgentHasStatusRunning(instanceDetails.serviceName);
+    // Waiting for metrics to start hitting PMM-Server
+    I.wait(20);
+  },
+).retry(1);
+
+Data(instances).Scenario(
+  'Verify dashboard after Remote GC Instances are added @not-ui-pipeline @gcp',
+  async ({
+    I, dashboardPage, adminPage, current,
+  }) => {
+    const {
+      instance, instanceType,
+    } = current;
+
+    const instanceDetails = getInstance(instance);
+
+    I.wait(10);
+    if (instanceType === 'mysql') {
+      I.amOnPage(dashboardPage.mySQLInstanceOverview.url);
+    }
+
+    if (instanceType === 'postgresql') {
+      I.amOnPage(dashboardPage.postgresqlInstanceOverviewDashboard.url);
+    }
+
+    dashboardPage.waitForDashboardOpened();
+    adminPage.applyTimeRange('Last 12 hours');
+    await dashboardPage.applyFilter('Service Name', instanceDetails.serviceName);
+    adminPage.performPageDown(5);
+    await dashboardPage.expandEachDashboardRow();
+    adminPage.performPageUp(5);
+    await dashboardPage.verifyThereAreNoGraphsWithNA();
+    await dashboardPage.verifyThereAreNoGraphsWithoutData(1);
+  },
+).retry(2);
+
+// skipping mysql gc because of PMM-9389
+Data(instances.filter((instance) => instance.instanceType.indexOf('mysql') === -1)).Scenario(
+  'Verify QAN after remote Google Cloud instance is added @not-ui-pipeline @gcp',
+  async ({
+    I, qanOverview, qanFilters, qanPage, current,
+  }) => {
+    const {
+      instance,
+    } = current;
+
+    const instanceDetails = getInstance(instance);
+
+    I.amOnPage(qanPage.url);
+    qanOverview.waitForOverviewLoaded();
+    qanFilters.waitForFiltersToLoad();
+    await qanFilters.applyFilter(instanceDetails.serviceName);
+    qanOverview.waitForOverviewLoaded();
+    const count = await qanOverview.getCountOfItems();
+
+    assert.ok(count > 0, `The queries for service ${instanceDetails.serviceName} instance do NOT exist`);
+  },
+).retry(1);
+
+Data(instances).Scenario(
+  'Check metrics from exporters are hitting PMM Server @not-ui-pipeline @gcp',
+  async ({ I, grafanaAPI, current }) => {
+    const {
+      instance, metric,
+    } = current;
+
+    const instanceDetails = getInstance(instance);
+
+    I.wait(10);
+    const response = await grafanaAPI.checkMetricExist(metric, { type: 'service_name', value: instanceDetails.serviceName });
+    const result = JSON.stringify(response.data.data.result);
+
+    assert.ok(response.data.data.result.length !== 0, `Metrics ${metric} from ${instanceDetails.serviceName} should be available but got empty ${result}`);
+  },
+);
+
+Scenario(
+  'Setting back to default Scrape Interval @not-ui-pipeline @gcp',
```

Comment:
This one to After block?

Reply:
the same reason, I did this intentionally to be able to read this from report